MyHashMap.java

```diff
@@ -0,0 +1,101 @@
+class MyHashMap {
+
+    static class Node 
+    {
+        int key, value;
+        Node next;
+        public Node (int key, int value)
+        {
+            this.key = key;
+            this.value = value;
+            next = null;
+        }
+    }
+    Node [] nodeArr = new Node[1000];
+        
+    /** Initialize your data structure here. */
+    public MyHashMap() {
+       
+    }
+    
+    /** value will always be non-negative. */
+    public void put(int key, int value) {
+        int index = key%1000;
+        Node insert = new Node(key, value);
+        if (nodeArr[index] == null)
+        {
+            nodeArr[index] = insert;
+            return;
+        }
+        if (nodeArr[index].key == key)
+        {
+            nodeArr[index].value = value;
+            return;
+        }
+        Node curr = nodeArr[index];
+        while(curr.next != null)
+        {
+            if (curr.next.key == key)
+            {
+                curr.next.value = value;
```

Comment:
If value is updated, the function should return, otherwise line 42 will also execute creating an unwanted duplicate node.